packages/wp-story-editor/src/components/checklist/accessibility/videoOptimizationCheckbox/index.js

```diff
@@ -37,6 +43,17 @@ const CheckboxContainer = styled.div`
   align-items: flex-start;
 `;
 
+const ButtonContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const BoldText = styled.span`
+  color: ${({ theme }) => theme.colors.standard.white};
+  font-weight: ${({ theme }) => theme.typography.weight.bold};
```

Comment:
nit, the `Text` component is adaptable to be nested, it's kinda 6 or half dozen, just mentioning it. 

```suggestion
const BoldText = styled(Text).attrs({ as: 'span', isBold: true })`
  color: ${({ theme }) => theme.colors.standard.white};`
```

Reply:
Ah sweet, thanks!